pkg/controller/v1alpha2/trial/trial_controller.go

```diff
@@ -245,6 +240,9 @@ func (r *ReconcileTrial) reconcileJob(instance *trialsv1alpha2.Trial, desiredJob
 	err = r.Get(context.TODO(), types.NamespacedName{Name: desiredJob.GetName(), Namespace: desiredJob.GetNamespace()}, deployedJob)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			if instance.IsCompleted()  {
```

Comment:
Which condition is being targeted here?

Reply:
```
func (trial *Trial) IsCompleted() bool {
	return trial.IsSucceeded() || trial.IsFailed() || trial.IsKilled()
}
```

Reply:
Is it the case that Get returns a stale entry after trial is completed?

Reply:
https://github.com/kubeflow/katib/blob/master/pkg/controller/v1alpha2/trial/trial_controller.go#L149 returns if instance is completed.

Reply:
do you mean add a comment like `IsCompleted() targets "IsSucceeded() || IsFailed() || IsKilled()"` everywhere IsCompleted() is called?

Reply:
Line 147~151 deleted in this PR

Reply:
Sorry. I missed that change.  Can you explain  the difference? And what I meant, is to add a comment why this check is added.  Readers will find it easier to follow the code.